File: app/src/test/java/com/github/goldy1992/mp3player/dagger/modules/MockAlbumArtPainterModule.java
package com.github.goldy1992.mp3player.dagger.modules;

import com.github.goldy1992.mp3player.client.AlbumArtPainter;

import dagger.Module;
import dagger.Provides;

import static org.mockito.Mockito.mock;

@Module
public class MockAlbumArtPainterModule {


    @Provides
    public AlbumArtPainter provideAlbumArtPainter() {
        return mock(AlbumArtPainter.class);
    }
}

File: client/testsupport/src/main/java/com/github/goldy1992/mp3player/client/testsupport/dagger/modules/MockAlbumArtPainterModule.kt
package com.github.goldy1992.mp3player.client.testsupport.dagger.modules

import com.github.goldy1992.mp3player.client.AlbumArtPainter
import dagger.Module
import dagger.Provides
import org.mockito.Mockito

@Module
class MockAlbumArtPainterModule {
    @Provides
    fun provideAlbumArtPainter(): AlbumArtPainter {
        return Mockito.mock(AlbumArtPainter::class.java)
    }
}